src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+const man = 'm';
```

Comment:
Consider using more descriptive constant names like 'male' instead of 'man' to improve code readability.